data/h3_data_importer/delete_h3_tables.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+import click
+import psycopg
+
+from utils import get_connection_info
+
+log = logging.getLogger("delete_h3_tables")
+logging.basicConfig(level=logging.INFO)
+
+
+@click.command()
+@click.option("--drop-contextuals", is_flag=True)
+@click.option("--dry-run", is_flag=True)
```

Comment:
Where is this dry-run set to False, so the code actually cleans the DB?

I am probably missing something, but what is the purpose of those flags in the actual code? Wouldn't be easier just to add some tests? This is a good feature to add tests

Reply:
For context, `click` is the lib used for cli stuff. When an option is set as a flag it is only set to true when it is present as a argument in the command line. The absence of `--dry-run`  in the _cli_ will make `dry_run` eval to `False` in the code.